cli/src/pcluster/api/flask_app.py

```diff
@@ -46,51 +48,96 @@ class ParallelClusterFlaskApp:
     def __init__(self, swagger_ui: bool = False, validate_responses=False):
         options = {"swagger_ui": swagger_ui}
 
-        app = connexion.FlaskApp(__name__, specification_dir="openapi/", skip_error_handlers=True)
-        app.app.json_encoder = encoder.JSONEncoder
-        app.add_api(
+        self.app = connexion.FlaskApp(__name__, specification_dir="openapi/", skip_error_handlers=True)
```

Comment:
minor: what about adding another name for this self.app? To avoid having self.app.app

Reply:
makes sense. let me do this as part of the next PR